src/components/ChosenUs/ChosenUs.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import { FC } from 'react';
+import { StContainerInner, StList, StTitle } from './styled';
+import { ListItem } from './ListItem/ListItem';
+import { DataArray } from './typing';
+
+interface TChosenProps {
+  data: DataArray;
+}
+
+export const ChosenUs: FC<TChosenProps> = ({ data }) => {
+  return (
+    <section>
+      <StContainerInner>
+        <StTitle>Нас выбирают более 1000 клиентов по всей России</StTitle>
+        <StList>
+          {data.map(item => {
+            return <ListItem key={item.id} icon={item.icon} text={item.text} />;
```

Comment:
Можно `index` использовать, чтобы в перебираемом объекте не добавлять id
```suggestion
            return <ListItem key={`item-${index}`} icon={item.icon} text={item.text} />;
```